cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -17,53 +15,232 @@
+                writeJSONfile(json, fileName);
+                System.out.println("SBOM: " + json);
+            } break;
+
+            case "addMetadata": {                             //This is Metadata Component --> name
+                Bom bom = readJSONfile(fileName);
+                bom = addMetadata(bom, name);
+                String json = generateBomJson(bom);
+                writeJSONfile(json, fileName);
+                System.out.println("SBOM: " + json);
+            } break;
 
-        Bom bom = createTestBom();
-        String json = generateBomJson(bom);
+            case "addMetadataProperty": {                     //This is MetaData--> Component --> Property -> name-value:
+                Bom bom = readJSONfile(fileName);
+                bom = addMetadataProperty(bom, name, value);
+                String json = generateBomJson(bom);
+                writeJSONfile(json, fileName);
+                System.out.println("SBOM: " + json);
+            } break;
 
-        System.out.println("SBOM: " + json);
+            case "addComponent": {                              //This adds Component with component name
+                Bom bom = readJSONfile(fileName);
+                bom = addComponent(bom, name);
+                String json = generateBomJson(bom);
+                writeJSONfile(json, fileName);
+                System.out.println("SBOM: " + json);
+            } break;
+
+            case "addComponentProp": {                             //This adds Components with name-value pairs to List
+                Bom bom = readJSONfile(fileName);
+                bom = addComponentProperty(bom, name, value);
+                String json = generateBomJson(bom);
+                writeJSONfile(json, fileName);
+                System.out.println("SBOM: " + json);
+            } break;
+
+            case "addExtRef": {                                    //This adds external Reference
+                Bom bom = readJSONfile(fileName);
+                bom = addExternalReference(bom, url, hashes, comment);
+                String json = generateBomJson(bom);
+                writeJSONfile(json, fileName);
+                System.out.println("SBOM: " + json);
+            } break;
+
+            case "addComponentExtRef": {                                 //This adds external Reference to Component
+                Bom bom = readJSONfile(fileName);
+                bom = addComponentExternalReference(bom, hashes, url, comment);
+                String json = generateBomJson(bom);
+                writeJSONfile(json, fileName);
+                System.out.println("SBOM: " + json);
+            } break;
+            default: {
+                System.out.println("Please enter a command.");
+            }
+        }
     }
 
-    static Bom createTestBom() {
+    static Bom createBom(final String name, final String version) {        //Create SBOM, test.JSON file
+        System.out.println(bom.getBomFormat());
```

Comment:
this debug System.out.. line needs removing, as does not compile, bom not in scope.